src/DGtal/images/ImageContainerByHashTree.h

```diff
@@ -276,6 +276,19 @@ namespace DGtal
      */
     Range range() ;
 
+    /**
+     * Give access to the underlying container.
+     * @return a (might be const) reference to the container.
+    */
+    inline const Self & container() const { return *this; };
+    inline Self & container() { return *this; };
```

Comment:
Stupid question. Doxygen allows to comment one method and extend the comment to the next method if  they have the same name (but different signature) ?

Reply:
I thought yes, but I was wrong. This is the doxygen generated in from the same scenario in [PointVector::data()](https://github.com/DGtal-team/DGtal/blob/master/src/DGtal/kernel/PointVector.h#L974-L980)

![image](https://user-images.githubusercontent.com/3021667/99707731-24c42b00-2a9d-11eb-9d9d-df7401704f88.png)

There is a `1/2`, `2/2` index in the upper corner, but the second method is not documented.
I will fix it.

Reply:
However there is  the command `@copydoc data()` that should work.

Reply:
Added @copydoc to the second function.